Fix link to the getting started docs

herflis opened the pull request.

B3zo0 commented on apps/sensenet/src/services/PersonalSettings.ts:
```diff
@@ -255,7 +255,7 @@ export const defaultSettings: PersonalSettingsType = {
         settings: {
           content:
-            '<div>To get started with sensenet</div><br /><a style="color: #26a69a; line-height: 2rem" href="https://community.sensenet.com/docs/concepts/overview">Overview</a><br /><a style="color: #26a69a; line-height: 2rem" href="https://community.sensenet.com/docs/tutorials/getting-started">Getting started</a><br /><a href="https://community.sensenet.com/tutorials" style="color: #26a69a; line ...
+            '<div>To get started with sensenet</div><br /><a style="color: #26a69a; line-height: 2rem" href="https://community.sensenet.com/docs/concepts/overview">Overview</a><br /><a style="color: #26a69a; line-height: 2rem" href="https://community.sensenet.com/docs/getting-started">Getting started</a><br /><a href="https://community.sensenet.com/tutorials" style="color: #26a69a; line-height: 2 ...
```
I think it would make sense to open these links in new tab.

herflis replied:
👍

B3zo0 commented:
I see 404 for [Overview](https://community.sensenet.com/docs/concepts/overview) this could be https://community.sensenet.com/docs/concepts maybe? also 404 for example apps.

herflis commented:
@B3zo0 I've changed the link you've mentioned first to the overview on  sensenet.com and till the example apps are not the part of the monorepo, the related article is not published so I've changed the link to a '\'.
